build.gradle

```diff
@@ -1380,7 +1409,9 @@ project(':clients') {
   }
 
   jar {
+    enabled false
     dependsOn createVersionFile
+    dependsOn 'shadowJar'
     from("$buildDir") {
         include "kafka/$buildVersionFileName"
     }
```

Comment:
can we make sure this build version file is still included in the final jar like before?

Reply:
Thanks for pointing that, added other files too.